.github/workflows/ci.yaml

```diff
@@ -397,6 +398,9 @@ jobs:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
       - name: clippy x64
         run: cargo clippy -p mirrord -p mirrord-layer -p mirrord-sip --target=x86_64-apple-darwin --tests -- -Wclippy::indexing_slicing -D warnings
+      # this is required for clippy lint because the layer path variable is checked at compile time
+      - name: create fake layer file for arm 64
```

Comment:
I think it'd be better to just add clippy ignore to that line with the warning..

Reply:
Oh actually i dont think this will work since its a compilation error rather than a clippy failure - if the var isnt given, that varibale wont have a value